src/cpp.cc

```diff
@@ -2517,8 +2517,14 @@ void Printer::Cpp::print_value_pp_tree(const AST &ast) {
   dec_indent();
   stream << indent() << "} else {" << endl;
   inc_indent();
-  stream << indent() << "out << res << '\\n';" << endl;
-  stream << indent() << "file_stream << res << '\\n' << std::endl;" << endl;
+  stream << indent() << "out << res;" << endl;
+  stream << indent() << "file_stream << \"\\\\documentclass[tikz, border=1mm]{standalone} \" << std::endl;" << endl;
+  stream << indent() << "file_stream << \"\\\\begin{document} \" << std::endl;" << endl;
+  stream << indent() << "file_stream << \"\\\\begin{tikzpicture}\" << std::endl;" << endl;
+  stream << indent() << "file_stream << \"\\\\node {root} \" << std::endl;" << endl; // Each Root generates a subplot; Need to name them individually so they can be generated all in one file. Should be possible since the root name won't need any senseful name
+  stream << indent() << "file_stream << res;" << endl; // TO-DO -> split res on "\n"(?) so we can generate a node entry for each entry in res
```

Comment:
Is there any way for me to manipulate the input of res? 
If i'm able to get all separate lines of res i should be able to generate subplots in one file for all trees. But since this is a List<Rope> i dont know how to properly access the contents. So far im trying to find a function that i could use inside of rtlib/rope.hh, so far without any progress.